universal-login-relayer/test/helpers/WalletCreator.ts

```diff
@@ -63,6 +63,7 @@ export class WalletCreator {
       signature,
       applicationInfo: TEST_APPLICATION_INFO
     });
+    await sleep(1000); // TODO: check status and wait for 'Success' state
```

Comment:
Will address in a follow-up PR

Reply:
You can use waitForDeploymentStatus from previous file